modules/standard/SysBasic.chpl

```diff
@@ -98,227 +175,757 @@ inline proc c_calloc(type eltType, size: integral) {
+
+/* Bad address. The system detected an invalid address in attempting to
+   use an argument of a call. (POSIX.1)
+ */
 extern const EFAULT:err_t;
+
+/* File too large. The size of a file exceeded the maximum. (POSIX.1) */
 extern const EFBIG:err_t;
+
+/* Host is down. A socket operation failed because the destination host was
+   down. (linux, FreeBSD)
+ */
 extern const EHOSTDOWN:err_t;
+
+/* No route to host. A socket operation was attempted to an unreachable host.
+   (POSIX.1)
+ */
 extern const EHOSTUNREACH:err_t;
+
+/* Identifier removed.  An IPC identifier was removed while the current
+   process was waiting on it.  (POSIX.1)
+ */
 extern const EIDRM:err_t;
+
+/*
+   Illegal byte sequence.  While decoding a multibyte character the function
+   came along an invalid or an incomplete sequence of bytes or the given wide
+   character is invalid.
+
+   This error might be returned for example in the case of an illegal UTF-8
+   byte sequence. (POSIX.1, C99)
+ */
 extern const EILSEQ:err_t;
+
+/* Operation now in progress.   An operation that takes a long time to complete
+   (such as a connect system call) was attempted on a non-blocking object.
+   (POSIX.1)
+  */
 extern const EINPROGRESS:err_t;
+
+/* Interrupted system call.  An asynchronous signal (such as SIGINT or SIGQUIT)
+   was caught by the process during the execution of an interruptible function.
+   If the signal handler performs a normal return, the interrupted system call
+   will seem to have returned the error condition. (POSIX.1)
+*/
 extern const EINTR:err_t;
+
+/* Invalid argument.  Some invalid argument was supplied.  (For example,
+   specifying an undefined signal to a signal system call or a kill system
+   call). (POSIX.1)
+*/
 extern const EINVAL:err_t;
+
+/* Input/output error.  Some physical input or output error occurred.  This
+   error will not be reported until a subsequent operation on the same file
+   descriptor and may be lost (over written) by any subsequent errors. (POSIX.1)
+  */
 extern const EIO:err_t;
+
+/* Socket is already connected.  A connect system call was made on an already
+   connected socket; or, a sendto or sendmsg system call on a connected socket
+   specified a destination when already connected. (POSIX.1)
+*/
 extern const EISCONN:err_t;
+
+/* Is a directory. An attempt was made to open a directory with write mode
+   specified.  (POSIX.1)
+ */
 extern const EISDIR:err_t;
+
+/* Is a named type file (linux only) */
 extern const EISNAM:err_t;
+
+/* Key has expired (linux only) */
 extern const EKEYEXPIRED:err_t;
+
+/* Key was rejected by service (linux only) */
 extern const EKEYREJECTED:err_t;
+
+/* Key has been revoked (linux only) */
 extern const EKEYREVOKED:err_t;
+
+/* Level 2 halted (linux only) */
 extern const EL2HLT:err_t;
+
+/* Level 2 not synchronized (linux only) */
 extern const EL2NSYNC:err_t;
+
+/* Level 3 halted (linux only) */
 extern const EL3HLT:err_t;
+
+/* Level 3 halted (linux only) */
```

Comment:
They're linux only, and the linux man page gives them the same description. So, at least, it's intentional from me...